comp/host-profiler/bundle.go

```diff
@@ -0,0 +1,24 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2025-present Datadog, Inc.
+
+//go:build hostprofiler
```

Comment:
Similarly for the component, it should only be imported by the host-profiler binary, so no need for a build tag ?